.gitignore

```diff
@@ -9,7 +9,7 @@
 *.idea
 
 # Build
-packages/Debian/build/
+packages/build/
```

Comment:
readme.rst says `packages/Debian/build/`

Reply:
Fixed `packages/build` in make_deb script too and this is the right path across all plugins.